test/tests/misc/multiple-nl-systems/restart_problem.i

```diff
@@ -0,0 +1,77 @@
+[Mesh]
+  file = 'starting_problem_out.e'
+[]
+
+[Problem]
+  nl_sys_names = 'u v'
+  allow_initial_conditions_with_restart = true
+[]
```

Comment:
```suggestion
  solve = false
[]
```
then you can use the same exodus

Reply:
(with a symlink)

Reply:
I had a thought. Right now the only real purpose of starting_problem.i is to create an output file that can be used as a starting point for restart_problem.i. What if instead I have only one test that starts from an output file in the gold folder, does the copying but doesn't run, and then I can just have a check to make sure nothing changed during the restart process?

Reply:
this would work I agree